repondeur/zam_repondeur/writer.py

```diff
@@ -26,8 +26,8 @@
     "num": "Num amdt",
     "auteur": "Auteur(s)",
     "date_depot": "Date de dépôt",
-    "discussion_commune": "Discussion commune ?",
-    "identique": "Identique ?",
+    "id_discussion_commune": "Discussion commune ?",
```

Comment:
I think we should change these two labels.

Reply:
Ideas?

Reply:
Identifiant discussion commune / Identifiant identique ?